packages/buildcop/src/buildcop.ts

```diff
@@ -614,13 +611,25 @@ buildcop.issueComparator = (
   a: Octokit.IssuesListForRepoResponseItem,
   b: Octokit.IssuesListForRepoResponseItem
 ) => {
+  if (a.state === 'open' && b.state !== 'open') {
+    return -1;
+  }
+  if (a.state !== 'open' && b.state === 'open') {
+    return 1;
+  }
+  // The issues are either both open or both not-open.
   if (buildcop.isFlaky(a) && !buildcop.isFlaky(b)) {
     return -1;
   }
   if (!buildcop.isFlaky(a) && buildcop.isFlaky(b)) {
     return 1;
   }
-  return a.number - b.number;
+  const aClose = parseClosedAt(a.closed_at);
+  const bClose = parseClosedAt(b.closed_at);
+  if (aClose && bClose) {
```

Comment:
Just be careful about truthiness here.  This will evaluate false for undefined, null, and 0.  That may be fine since it's a date :)  I tend to prefer more strict checks.

Reply:
That seems OK for this particular case. This check would not be valid for any of those values. I suppose we know that the result is `number|undefined`, though.

I'm open here. Will defer to you for JS style.

Reply:
Nah this is fine, just shining light on it :)